src/PostgREST/App.hs

```diff
@@ -72,7 +72,7 @@ app conf reqBody req =
                   . limitT range
                   . orderT (orderParse qq)
                   . whereT qt qq
-                  $ selectStar qt
+                  $ selectT qt qq
```

Comment:
I've been using the "T" ending to mean that the function transforms one `PStmt` into another. Since `selectT` creates a statement, maybe just call it `select`.